upgrade/php/ps_remove_controller_tab.php

```diff
@@ -28,7 +28,7 @@ function ps_remove_controller_tab($className, $quickAccessUrls = [])
     // delete orphan tab langs
     Db::getInstance()->execute(
         sprintf(
-            'DELETE FROM `%stab_lang` WHERE `id_tab` NOT IN (SELECT id_tab FROM `%s_tab`)',
+            'DELETE FROM `%stab_lang` WHERE `id_tab` NOT IN (SELECT id_tab FROM `%stab`)',
```

Comment:
About this bug, there is already an opened PR from @Prestaworks  https://github.com/PrestaShop/autoupgrade/pull/559

Reply:
Yes it's a full pack with all the currently detected issue using a DB dump from a pretashop 1.7.4.x :)